### DISCUSSION TITLE
Templates-document manager

### DISCUSSION THREAD

**Discussion by slaterha** [Budibase/budibase#8359] [Help]
We recently have switched databases and needing a reconnection. I can't see where certain fields may be affected by this and was trying to see if i could get some direction on this as our developer that created it is out

**Comment by melohagan** [comment]
The database credentials would be set on the datasource config pages within the **Data**section.

E.g.
![Screenshot 2022-10-24 at 16 44 22](https://user-images.githubusercontent.com/101575380/197568438-dfc97caf-b948-4b2f-a62a-710d4f16289a.png)
